### PR TITLE
renamed x to meaningful variable in resnet.py

### PR DESCRIPTION
**Pull request by i-am-epic**
Hello @patil-suraj can you verify it
Thanks

### CODE REVIEW

**Comment by i-am-epic** [comment]
Black says : Cannot parse: 100:34:     def __init__(self, channels=N one, out_channels=None, use_conv=False, fir_kernel=(1, 3, 3, 1)):
but I have not modified those lines

**Comment by patrickvonplaten** [comment]
Hey @i-am-epic - could you try updating your black version? 

Re-opening the PR :-)

**Review by i-am-epic — COMMENTED** [review]
renamed variables x and p

**Comment by i-am-epic** [comment]
This was my first Contribution, 
any more changes let me know,
Thank you

File: src/diffusers/models/resnet.py [hunk 1]
```diff
@@ -34,21 +34,21 @@ def __init__(self, channels, use_conv=False, use_conv_transpose=False, out_chann
         else:
             self.Conv2d_0 = conv
 
-    def forward(self, hidden_states):
-        assert hidden_states.shape[1] == self.channels
+    def forward(self, input_tensor):
+        assert input_tensor.shape[1] == self.channels
```
Comment (patil-suraj) [inline src/diffusers/models/resnet.py:38]: This shouldn't be changed.

File: src/diffusers/models/resnet.py [hunk 2]
```diff
@@ -84,16 +84,16 @@ def __init__(self, channels, use_conv=False, out_channels=None, padding=1, name=
         else:
             self.conv = conv
 
-    def forward(self, hidden_states):
-        assert hidden_states.shape[1] == self.channels
+    def forward(self, input_tensor):
```
Comment (patil-suraj) [inline src/diffusers/models/resnet.py:87]: same comment as above.

**Review by patil-suraj — CHANGES_REQUESTED** [review]
Thanks a lot for the PR @i-am-epic ! Here we should only rename the single letter variables. Other names are well chosen so let's not change them. We should update the PR accordingly.

File: src/diffusers/models/resnet.py (hunk 1 above)
Comment (i-am-epic) [reply src/diffusers/models/resnet.py:38]: Hey @patil-suraj,
Thanks for pointing out the changes
those variables were recently changed in PR #676  
ill try to rename variables accordingly.

**Comment by i-am-epic** [comment]
@patil-suraj can you verify the updated code
Thanks :)

**Review by patil-suraj — APPROVED** [review]
Looks good, thank you for working on this. Should be good to merge once the tests pass.

**Comment by i-am-epic** [comment]
Thanks for the guidance and help,
as this was my first contribution and PR, it helped me to learn a lot and this is just the beginning of my contributions.
:)

**Pull request merged**